doc/source/analytics/explainers.md

```diff
@@ -36,9 +36,9 @@ For Alibi explainers that need to be trained you should
 
 The runtime environment in our [Alibi Explain Server](https://github.com/SeldonIO/seldon-core/tree/master/components/alibi-explain-server) is locked using [Poetry](https://python-poetry.org/). See our e2e example [here](../examples/iris_explainer_poetry.html) on how to use that definition to train your explainers.
 
-### V2 protocol for explainer using [MLServer](https://github.com/SeldonIO/MLServer) (incubating)
+### Open Inference Protocol for explainer using [MLServer](https://github.com/SeldonIO/MLServer) (incubating)
```

Comment:
Is it incubating?

Reply:
@matthewlowdon  Please suggest what it should be. I just changed only the name in that line.

Reply:
Fixed now. Incorporated the suggestions that @lc525  made.